client/state/selectors/is-calypsoify-gutenberg-enabled.js

```diff
@@ -7,16 +7,20 @@ import { getSelectedEditor } from 'state/selectors/get-selected-editor';
 import isVipSite from 'state/selectors/is-vip-site';
 import { isJetpackSite } from 'state/sites/selectors';
 
-export const isCalypsoifyGutenbergEnabled = ( state, siteId ) => {
+export const isCalypsoifyGutenbergEnabled = ( state, siteId, { skipSelectedEditorCheck } = {} ) => {
 	if ( ! siteId ) {
 		return false;
 	}
 
 	const calypsoifyGutenberg = isEnabled( 'calypsoify/gutenberg' );
 	const isJetpack = isJetpackSite( state, siteId );
 	const isVip = isVipSite( state, siteId );
-	const selectedEditor = getSelectedEditor( state, siteId );
 
+	if ( skipSelectedEditorCheck ) {
```

Comment:
this is also smelling bad to me. We always call this with `true`, or not at all. And we only pass this down on `EditorGutenbergOptInDialog`. I'd rather have two selectors (that we can delete later) than an `options` object. We can even compose them for extra fancy functional programming.